l10n_it_dichiarazione_intento/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+Creare una posizione fiscale per le dichiarazioni d'intento, con relativa imposta (ad esempio "22% dichiarazione d'intento (debito)").
+Impostare la mappatura ad esempio tra "IVA al 22% (debito)" e "22% dichiarazione d'intento (debito)".
```

Comment:
@sergiocorato @labaggio @OpenCode in realtà qua andrebbe "fuori campo" invece di `22% dichiarazione d'intento`?

Reply:
Non esisteva un aliquota apposita. Non saprei dirti se Fuori Campo è quella corretta. Bisognerebbe sentire un parere contabile.

Reply:
Noi riceviamo fatture dai fornitori in esenzione IVA avendo emesso dichiarazione di intento e generalmente il codice IVA sulle fatture è: cod.Art. 8-8bis del D.P.R. n. 633/1972, Natura operazione: N3 (non imponibile)

Reply:
Grazie, modificato